### ISSUE TITLE
Contact icons too small

### ISSUE THREAD

**Issue by connorlanigan** [smallbusinesshero/dds-wvv-frontend#59]
The contact icons on the store detail pages are too small to be properly used.

Their size should be increased so that they are easily touchable (i.e. the touch target area should be at least 9mm, see [Google's web a11y guidelines](https://developers.google.com/web/fundamentals/accessibility/accessible-styles)).
